pkg/session/session.go

```diff
@@ -929,6 +929,21 @@ func (s *session) CommitTxn(ctx context.Context) error {
 		s.sessionVars.StmtCtx.MergeExecDetails(nil, commitDetail)
 	}
 
+	if err == nil {
+		// save CommitTS in sessionVars for invariant check
+		// TODO: enable LastCommitTS with a session variable
```

Comment:
I will add a session variable.

Reply:
Added in a separate PR https://github.com/pingcap/tidb/pull/57313, to make review easier.

Reply:
Per discussion in https://github.com/pingcap/tidb/pull/57313#discussion_r1856131594, we don't need a session variable.